README.md

```diff
@@ -1,2 +1,25 @@
 # translate
-A [maubot](https://github.com/maubot/maubot) to translate words.
+A [maubot](https://github.com/maubot/maubot) to translate words using google-tranlslator (DeepL is planned too)
+
+Invite the bot to your room, so you can ask it for translatins, example:
+
+    !translate en ru Hello world.
+    
+which results in
+
+    Translate Bot:
+    > rubo77
+    > !translate en ru Hello world.
+    Привет, мир.
+    
+supported_languages: 
```

Comment:
https://cloud.google.com/translate/docs/languages

Reply:
what do you want to say with that link?

Reply:
Yes, it mostly just uses Google Translate directly